tests/c_languages/CPPCheckBearTest.py

```diff
@@ -14,22 +14,36 @@ def get_absolute_test_path(file):
 
 
 @generate_skip_decorator(CPPCheckBear)
-class CPPCheckBearTest(LocalBearTestHelper):
+class CPPCheckBearTest(unittest.TestCase):
 
     def setUp(self):
         self.section = Section('cppcheck')
-        self.uut = CPPCheckBear(self.section, Queue())
-        self.good_file = get_absolute_test_path('good_file.cpp')
-        self.bad_file = get_absolute_test_path('bad_file.cpp')
-        self.warn_file = get_absolute_test_path('warn_file.cpp')
-
-    def test_default(self):
-        self.check_validity(self.uut, [], self.good_file)
-        self.check_invalidity(self.uut, [], self.bad_file)
-        self.check_validity(self.uut, [], self.warn_file)
-
-    def test_enable(self):
         self.section.append(Setting('enable', 'unusedFunction'))
-        self.check_validity(self.uut, [], self.good_file)
-        self.check_invalidity(self.uut, [], self.bad_file)
-        self.check_invalidity(self.uut, [], self.warn_file)
+        self.file_dict = {}
+        self.queue = Queue()
+        self.test_files = ['good_file.cpp', 'bad_file.cpp', 'warn_file.cpp']
+
+    def get_results(self, files_to_check):
+        files = [get_absolute_test_path(file) for file in files_to_check]
+        for filename in files:
+            with open(filename, 'r', encoding='utf-8') as _file:
+                self.file_dict[filename] = tuple(_file.readlines())
```

Comment:
underscore prefixes usually only occur in classes for members that are meant to be "protected". Please don't use the underscore as a prefix for local variables :+1: